Sources/Comment.swift

```diff
@@ -0,0 +1,124 @@
+//
+//  Comment.swift
+//  SwiftStack
+//
+//  Created by FelixSFD on 07.12.16.
+//
+//
+
+import Foundation
+
+/**
+ Represents a comment to a `Post`.
+ 
+ - author: FelixSFD
+ 
+ - seealso: [StackExchange API](https://api.stackexchange.com/docs/types/comment)
+ */
+public class Comment: JsonConvertible {
```

Comment:
Can we make `Comment` a subclass of `Post`?  If not directly, can we make `Comment` and `Post` both subclasses of a parent class that contains the properties used by both?